packages/components/src/components/hds/copy/snippet/index.ts

```diff
@@ -102,23 +132,25 @@ export default class HdsCopySnippetIndexComponent extends Component {
   }
 
   @action
-  onSuccess(args) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  onSuccess(args: any) {
```

Comment:
same as for the `Copy::Button`
```suggestion
  onSuccess(args: HdsClipboardModifierSignature['Args']['Named']['onSuccess']) {
```